User-Help/GlobalFAQ.md

```diff
@@ -0,0 +1,34 @@
+<!--- Codidact FAQ -->
+
+# Codidact FAQ
+
+## What is Codidact?
+
+Codidact is an open-source initiative to provide a high-quality, community-controlled, open source Q&A platform. Visit <https://codidact.org> to learn more about the platform we're building to empower anyone to host a Q&A community, or visit <https://codidact.com> to browse our family of hosted sites. 
+
+## Who runs Codidact?
+
+Users just like you! The software is built and hosted by volunteers who are passionate about the need for a community-driven Q&A site. Each site is moderated by a team of dedicated, experienced users who spend their time weeding out spam, coaching newbies, and ensuring questions are on-topic and high-quality. 
+
+## How do I ask a question?
+
+You will need to register an account. Make sure you are logged into the appropriate topic-related site for your question and click the "Ask Question" button at the top of the page. A template should appear with helpful guidelines on how to ask a quality question. Note that some communities may require you to have a certain trust level before asking a question; always check out the local FAQ page for important information before contributing.
+
+## What are trust levels and how are they calculated?
+<!-- I'm omitting any discussion of reputation for now since consensus seems to be that it is beyond MVP. If that is implemented we can add information here. -->
+Trust levels are the system Codidact uses to determine user privileges. Everyone starts at Trust Level 0 and gradually gains the ability to contribute at a higher level. Trust levels are earned by positively contributing to the site. For more information, see <https://github.com/codidact/docs/wiki/User-Privileges>. 
```

Comment:
Ultimately we'll want doc to be on-site, not on our GitHub wiki, but we don't have that yet.  Can you please file an issue (for later) to sweep the help for github links and replace them with links to help topics (that we will have to create)?  I don't want to lose track of it but we don't need to address it now.  Thanks.

Reply:
Yes, I actually have a trust level page on the project board, but it seems this is sufficiently in-flux that we can use the wiki as a source of truth for now and later do something more comprehensive on reputation / trust levels / privileges / etc. Will add issue to review links later.